osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -55,10 +57,14 @@ public override void OnResuming(IScreen last)
             if (removeAutoModOnResume)
             {
                 var autoType = Ruleset.Value.CreateInstance().GetAutoplayMod()?.GetType();
+                var cinemaType = Ruleset.Value.CreateInstance().GetAllMods().OfType<ModCinema>().FirstOrDefault()?.GetType();
 
                 if (autoType != null)
                     ModSelect.DeselectTypes(new[] { autoType }, true);
 
+                if (cinemaType != null)
+                    ModSelect.DeselectTypes(new[] { cinemaType }, true);
```

Comment:
I would be in favour of storing the selected mod instead of the `removeAutoModOnResume` flag, and then using that to remove it on resume.